README.md

```diff
@@ -1,80 +1,68 @@
-# ubuntu-advantage tool
+# Ubuntu Advantage Client
 
-[![Build Status](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-script.svg?branch=master)](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-script)
+[![Build Status](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-client.svg?branch=master)](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-client)
 
-This tool is used to enable or disable specific Ubuntu Advantage offerings from Canonical. Version 18.1 or later is a python-based client which talks to a
-UA Contract backend service API to navigate UA contracts, entitlement details
-and status.
+The Ubuntu Advantage client provides users with a simple mechanism to
+view, enable, and disable offerings from Canonical on their system. The
+following entitlements are supported:
 
-Currently it supports setup and maintenance of the following entitlements:
+- [Ubuntu Extended Security Maintenance](https://ubuntu.com/esm)
+- [Livepatch Service](https://www.ubuntu.com/livepatch)
+- FIPS 140-2 Certified Modules
+- FIPS 140-2 Non-Certified Module Updates
+- Common Criteria EAL2 certification artifacts provisioning
 
-- [Ubuntu Extended Security Maintenance](https://ubuntu.com/esm) archive.
-- [Canonical Livepatch](https://www.ubuntu.com/server/livepatch) service for managed live kernel patching.
-- Canonical FIPS 140-2 Certified Modules. Install Configure and Enable FIPS modules.
-- Canonical Common Criteria EAL2 certification artifacts provisioning
-
-Run
-
-``
-$ [sudo] python -m uaclient.cli --help
-```
-
-to display usage information.
+## Obtaining the Client
 
+The client comes pre-installed on all Ubuntu systems. Users can run the
+`ua` command to learn more or view the manpage.
 
 ## Testing
 
-System tests and tests lint:
+All unit and lint tests are run using tox:
 
-```
-$ tox
+```shell
+tox
 ```
 
-Lint:
+The client also includes built-in dep8 tests. These are run as follows:
 
+```shell
+autopkgtest --shell-fail . -- lxd ubuntu:xenial
```

Comment:
Due to our minimal lxc images we need minimally   to add a -U to apt update before test=

So, let's make this line 
autopkgtest -U --shell-fail . -- lxd ubuntu:xenial

The autopkgtests are currently broken, I've added issue #176 to track this issue.